test/specs/elements/Loader/Loader-test.js

```diff
@@ -12,24 +12,39 @@ describe('Loader', () => {
   common.propKeyOnlyToClassName(Loader, 'active')
   common.propKeyOnlyToClassName(Loader, 'disabled')
   common.propKeyOnlyToClassName(Loader, 'indeterminate')
-  common.propKeyOrValueAndKeyToClassName(Loader, 'inline')
   common.propKeyOnlyToClassName(Loader, 'inverted')
-  common.propKeyOnlyToClassName(Loader, 'text')
+
+  common.propKeyOrValueAndKeyToClassName(Loader, 'inline')
+
   common.propValueOnlyToClassName(Loader, 'size')
 
-  describe('text prop', () => {
-    it('renders children', () => {
+  describe('content', () => {
+    it('renders text', () => {
+      const text = faker.hacker.phrase()
+
+      shallow(<Loader content={text} />)
+        .should.contain.text(text)
+    })
+  })
+
+  describe('text (class)', () => {
```

Comment:
This can be replaced with `commonTest`, by I don't sure that it's good idea.

```jsx
 common.propKeyOnlyToClassName(Loader, 'required', {
    requiredProps: { children: faker },
  })
 common.propKeyOnlyToClassName(Loader, 'required', {
    requiredProps: { content: faker },
  })
```